content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -275,3 +279,15 @@ Such a controller is not a secure mechanism unless it not only verifies the Comm
 in the CSR but also verifies the requested IPs and domain names. This would prevent
 a malicious actor that has access to a kubelet client certificate to create
 CSRs requesting serving certificates for any IP or domain name.
+
+## Troubleshooting kubelet client certificates rotation {#kubelet-client-cert}
```

Comment:
we should move this to https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/